pkg/verifier/factory/factory.go

```diff
@@ -82,7 +82,7 @@ func CreateVerifierFromConfig(verifierConfig config.VerifierConfig, configVersio
 
 	verifierFactory, ok := builtInVerifiers[verifierNameStr]
 	if ok {
-		return verifierFactory.Create(configVersion, verifierConfig)
+		return verifierFactory.Create(configVersion, verifierConfig, pluginBinDir[0])
```

Comment:
one question: do we have the assumption that it's always the first?

Reply:
Yes we do currently. The `pluginBinDir` parameter is an array but we really only look for the first value. We should make it a single value instead of an array but this might be out of scope for this?